src/java/fr/paris/lutece/plugins/suggest/service/search/SuggestIndexer.java

```diff
@@ -269,11 +259,7 @@ public void indexDocuments( ) throws IOException, InterruptedException, SiteMess
         {
             new HtmlParser( ).parse( new ByteArrayInputStream( writerFieldContent.toString( ).getBytes( ) ), handler, metadata, new ParseContext( ) );
         }
-        catch( SAXException e )
-        {
-            throw new AppException( "Error during page parsing." );
-        }
-        catch( TikaException e )
+        catch( SAXException | TikaException e )
         {
             throw new AppException( "Error during page parsing." );
```

Comment:
Maybe as a drive by correction you should set the AppException Cause here to not loose the details of the error

Reply:
Ok, I will add the Exception "e" in the AppException's argument.